src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -625,6 +627,12 @@ def forward(
         output_hidden_states=None,
         return_dict=None,
```

Comment:
```suggestion
        return_dict=None,
        **kwargs,
```
I would remove `position_ids` from the forward and absorb it with `kwargs` and then do:

```python
if kwargs.pop("position_ids", None) is not None:
    warnigs.warn(...)
```

Reply:
Hum how is this superior? Introducing kwargs feels like opening a can full of worm? I mean we could just keep `position_ids` in the function argument and do nothing from it and signal that we don't use it? I've been meaning to introduce a decorator where we signal deprecate arguments if that makes sense to you.

Reply:
Guess it's not that big of a deal in the end, but again as I understand the whole PR about `position_ids` is that we are correcting a bug - not deprecating a working functionality, *i.e.* if people have passed `position_ids` before (which they very probably have **not** done), then it's just a no-op operation => so now removing this not used / useless function argument is completely fine for me. To be more or less 100% backward compatible, I think it's best if we add a `kwarg` that absorb `position_ids` as explained above because it makes it very clear from the doc string that `position_ids` doesn't exist anymore. If we leave as is, then `position_ids` is from the docs still an existing and functional function arg which it is not IMO

Reply:
Hum ... the advantage of keeping `position_ids` without introducing `**kwargs` is that forward will fail if you feed it anything else typically `potato=None` and doesn't if you use `kwargs`

Reply:
No, the kwargs should then be consumed: you pop `position_ids`, raise a deprecation warning if we want to, and then raise a ValueError if there is anything else left in the kwargs.

Reply:
Okay.